app/Models/User.php

```diff
@@ -568,6 +568,7 @@ public function getRelationshipPermissionMap(): array
             'rsvps' => 'rsvps',
             'roles' => 'roles-and-permissions',
             'permissions' => 'roles-and-permissions',
+            'assignments.travel' => 'users',
```

Comment:
`AuthorizeInclude` assumes the permission starts with `read-`, and there's just one permission for travel called `manage-travel`. I think it's fine to expose TAs to `read-users` but I'm open to other opinions on that.

I also do want to enforce eager-loading here.